app/views/styleguide/pages/index.html.erb

```diff
@@ -4,3 +4,14 @@
 <!-- Add new components here. -->
 <!-- You should create individual styleguide pages for them too. -->
 <br/>
+<%= react_component("OnboardingTopBar", props: {
+  translations: {"progress_label": "Marketplace progress","next_step": "Next"},
+  status: {
+    "community_id": 501,
+    "slogan_and_description": true,
+    "cover_photo": true,
+    "filter": true,
+    "paypal": false,
+    "listing": true,
+    "invitation": true}
+  }, prerender: false) %>
```

Comment:
I added props to the styleguide's pages_controller (as we do normally on Rails side)